corehq/apps/export/models/new.py

```diff
@@ -84,7 +84,7 @@ class ExportColumn(DocumentSchema):
     # A list of constants that map to functions to transform the column value
     transforms = ListProperty(validators=is_valid_transform)
 
-    def get_value(self, doc, base_path):
+    def get_value(self, doc, base_path, transform_dates=False):
```

Comment:
Yeah. I added it back to conform to Ben's latest PR: https://github.com/dimagi/commcare-hq/pull/10318